### DISCUSSION TITLE
GraphQL Single Type values are always null

### DISCUSSION THREAD

**Discussion by ijadams** [strapi/strapi#7345] [Legacy Topics]
Hey All - I'm using Strapi with Next.js and GraphQL. I cant seem to understand why my single type queries are returning null. I followed the restaurants example and the collections queries are working fine. Just having some trouble with Single types:

```
{
  restaurants {
    id
    name
  }
  homepage {
    test
  }
  articles {
    id
  }
}
```

Yields

```
{
  "data": {
    "restaurants": [
      {
        "id": "1",
        "name": "test2"
      }
    ],
    "homepage": {
      "test": null
    },
    "articles": [
      {
        "id": "1"
      },
      {
        "id": "2"
      }
    ]
  }
}
```

**Comment by derrickmehaffy** [comment]
Can you provide the model settings files used for your collection and single types so I can try to reproduce this?

**Reply by ijadams** [reply]
homepage.settings.json

```json
{
  "kind": "singleType",
  "collectionName": "homepages",
  "info": {
    "name": "Homepage"
  },
  "options": {
    "increments": true,
    "timestamps": true
  },
  "attributes": {
    "test": {
      "type": "integer"
    }
  }
}
```

**Reply by lauriejim** [reply]
I just tried with a fresh new app and it works well.
Make sure you fill data on your homepage.
